hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/DefaultS3ClientFactory.java

```diff
@@ -401,4 +411,20 @@ private static Region getS3RegionFromEndpoint(final String endpoint,
     return Region.of(AWS_S3_DEFAULT_REGION);
   }
 
+  private static <BuilderT extends S3BaseClientBuilder<BuilderT, ClientT>, ClientT> void
+      applyS3AccessGrantsConfigurations(BuilderT builder, Configuration conf) {
+    if (!conf.getBoolean(AWS_S3_ACCESS_GRANTS_ENABLED, false)){
```

Comment:
define and use a constant `AWS_S3_ACCESS_GRANTS_ENABLED` here.

makes it easier to see/change what the default is in future.